src/pluto/pluto.py

```diff
@@ -509,6 +508,42 @@ def create_expected(df, v1, v2):
                     st.markdown(f"* in {v2} but not in {v1}:")
                     st.write(in2not1)
 
+
+    def create_outlier(df, v1, condo, mapped):
+        outlier = df.loc[
+            (df.condo == condo)
+            & (df.mapped == mapped)
+            & (df.v == v1),
+            :,
+        ]
+
+        outlier_records = outlier.to_dict("records")
+        v1_out = [i["outlier"] for i in outlier_records if i["v"] == v1][0]
+
+        building_area_increase = [i["values"] for i in v1_out if i["field"] == 'building_area_increase'][0]
+        df1 = pd.DataFrame(building_area_increase)
+        version_pair=df1['pair'].unique()
+        df1=df1.drop(columns=['pair'])
+        df1['bbl']=pd.to_numeric(df1['bbl'], downcast='integer')
+        st.markdown(f'### Table of BBLs with Unreasonable Increase in Building Area {version_pair}')
+        st.dataframe(df1,900,200)
+        st.info('The table displays all BBLs where building area is more than doubled since previous version.')
+
+        st.markdown(f'### Table of BBLs with 50+ unitsres and resarea/unitsres < 300')
```

Comment:
To Sasha's point about who the audience for this is, wonder if we want to include any data dictionary type information here like what resarea and unitsres are, or why this ratio matters, or what the unit of 300 is

Reply:
right, totally agree. I am still not quite sure about why we need to select out these BBLs and what does the ratio mean.